Paper UI: build failure: integrity checksum failed

Travis CI failed: https://travis-ci.org/eclipse/smarthome/builds/355753272
Log: [log.txt](https://github.com/eclipse/smarthome/files/1828859/log.txt)

```text
[INFO] --- frontend-maven-plugin:1.0:npm (npm install) @ org.eclipse.smarthome.ui.paper ---
[INFO] Running 'npm install' in /home/travis/build/eclipse/smarthome/extensions/ui/org.eclipse.smarthome.ui.paper
[WARNING] npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[WARNING] npm WARN deprecated gulp-util@2.2.20: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[WARNING] npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
[WARNING] npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
[WARNING] npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[WARNING] npm WARN deprecated express@2.5.11: express 2.x series is deprecated
[WARNING] npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
[WARNING] npm WARN deprecated connect@1.9.2: connect 1.x series is deprecated
[WARNING] npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
[WARNING] npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[WARNING] npm WARN deprecated formidable@1.0.17: Old versions of Formidable are not compatible with the current Node.js; Upgrade to 1.2.0 or later
[WARNING] npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
[ERROR] npm ERR! code EINTEGRITY
[ERROR] npm ERR! sha1-qced3wuE72sv5IoNEVe0JWioH1o= integrity checksum failed when using sha1: wanted sha1-qced3wuE72sv5IoNEVe0JWioH1o= but got sha1-YaWyHl7atUePdG49lW2KdmjqbpU=. (2482734 bytes)
[ERROR] 
[ERROR] npm ERR! A complete log of this run can be found in:
[ERROR] npm ERR!     /home/travis/build/eclipse/smarthome/extensions/ui/org.eclipse.smarthome.ui.paper/npm_cache/_logs/2018-03-20T09_32_40_691Z-debug.log
```

* wanted `sha1-qced3wuE72sv5IoNEVe0JWioH1o=`
* got `sha1-YaWyHl7atUePdG49lW2KdmjqbpU=`

--- a/extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml
+++ b/extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml
@@ -57,7 +57,7 @@
 
         <configuration>
           <nodeVersion>v8.1.2</nodeVersion>
-          <npmVersion>5.0.3</npmVersion>
+          <npmVersion>5.8.0</npmVersion>
           <nodeDownloadRoot>http://nodejs.org/dist/</nodeDownloadRoot>
           <npmDownloadRoot>http://registry.npmjs.org/npm/-/</npmDownloadRoot>
           <environmentVariables>

